docs/src/architecture/transactions.md

```diff
@@ -1,61 +1,48 @@
 # Transactions
-Asset transfers between accounts are done by executing transactions. A transaction is always executed against a single account and causes a provable state-change. It can consume zero or more notes, and it may produce zero or more notes as shown in the diagram below.
+Asset transfers between accounts are done by executing transactions. Miden aims for parallel and private transaction execution. Therefore, a transaction is always performed against a single account which provides asynchronicity. And, every transaction causes a provable state-change and a STARK proof thereof. Thus, when executed locally, it provides privacy.
 
-It can be expressed as a state-transition function that takes an account and `0 to n` notes to map it to another version of that account and produces `0 to n` notes
+## Transaction design
+Transactions can be described as state-transition function T that takes an account and `0 to n` notes to map it to another version of that account and produces `0 to n` notes
 
 $
 T(A, \sum_{\substack{
-   0<i<n
+   0 \leq i < 1024
   }}
  N(i))) \to (A', \sum_{\substack{
-   0<j<m
+   0 \leq j < 4096
   }}
  N(j))
 $
-, where  $ A \in { Accounts }$, $N \in { Notes } $
+, where  $ A \in { Accounts }$, $N \in { Notes }, 0 < i + j  $
 
 It looks like
 
 <p align="center">
     <img src="../diagrams/architecture/transaction/Transaction.png">
 </p>
 
-In addition to specifying inputs and outputs, a transaction must also include an executable program. The transaction program has a well-defined structure which must do the following:
+### Transaction inputs and outputs
+As inputs, a transaction accepts the state of one single account and up to `1023` notes for consumption. The transaction's output will consist of that single account and up to `4096` newly created notes.
```

Comment:
Transactions also take an optional user defined transaction script which is executed after all input notes have been consumed.

Reply:
thanks, changed it